app/Track.php

```diff
@@ -93,6 +93,27 @@ class Track extends Model
         ],
     ];
 
+    public static $CacheableFormats = [
```

Comment:
This array duplicates a ton of data from `Track::$Formats`. Is any of it other than the format names actually being used? It would be good to strip this down to a simple array of format names and refer to `Track::$Formats` for any of the other information.